main/modes/lumberjack/lumberjack.c

```diff
@@ -173,7 +172,8 @@ static void lumberjackEnterMode(void)
 
 static void lumberjackLoadSave(void)
 {
-    size_t len = sizeof(lumberjack->save);
+    size_t len = sizeof(lumberjackUnlock_t);
```

Comment:
You should still reset this variable (or not use it) on this later line:
```
writeNvsBlob(LUMBERJACK_SAVE, &lumberjack->save, len);
```
If there's nothing to read, this may be 0, and writing 0 byes isn't right.

Reply:
This should be resolved with [8c9b799d] do I have to do a new pull request for this?

Reply:
Nope, if you add more commits to the same branch (`lumberjacks-bug-fixes`) they get automatically added to this PR

Reply:
You still have this on line 237 though